crawl-ref/source/dat/descript/branches.txt

```diff
@@ -102,21 +102,29 @@ Dis
 The Iron City of Dis is home to many of the most resilient guardians of hell.
 Those who do not belong in the halls of Dis will find the very earth itself
 opposing them.
```

Comment:
These descriptions feel a little weird now. There's:
- Overall flavor
- Hell effects? (Which is a mechanical description, but a very vague one)
- Separately, the hell passive.

And both the timed and passive hell effects are themed in a sort of overlapping way.

This is not that huge a deal, mostly because I think few people actually read these descriptions, but it feels like there must be some better way to explain what happens here. Maybe being more explicit about hell effects would help..?